_posts/2023-07-20-alert-anomaly-visual.md

```diff
@@ -0,0 +1,136 @@
+---
+layout: post
+title: "Overlaying anomalies and alerts on OpenSearch Dashboards visualizations"
+authors: 
+  - ashiasgr
```

Comment:
I would not recommend listing 11 authors for the post. Rather than listing everyone who worked on the feature, which is what I assume we've done here, let's just list actual contributors to the blog post.